remote_function/udf_server.py

```diff
@@ -1,15 +1,12 @@
 from flask import Flask, request, jsonify, send_file, after_this_request
 import cv2
-import numpy as np
 import json
 from datetime import datetime, timezone
 import os
 import sys
-from collections import defaultdict, deque
-import skvideo.io
-import imutils
 import uuid
 from zipfile import ZipFile
+from werkzeug.utils import secure_filename
```

Comment:
Is this 3rd party utility safe to include as a part of the merge? If its already part of the requirements, yes, but otherwise we might need additional updates to make sure we dont break anything, add new licensing requirements, etc.

Reply:
The utility is automatically installed with Flask and has the same licensing requirements. We have already included Flask in the requirements. The secure_filename is the recommended way of addressing filename security issues in Flask-based servers.

Reply:
Alright, as long as we're not adding additional licensing, scanning requirements at the moment I'm going to approve.